proc_Hahn_echoph.py

```diff
@@ -7,13 +7,13 @@
 
 
 signal_pathway = {"ph1": 1, "ph2": 0}
-with psd.fl_mod() as fl:
+with psd.figlist_var() as fl:
     for searchstr, exp_type, nodename, postproc, label_str, slice_f in [
         (
             "200302_alex_probe_water",
-            "test_equip",
+            "ODNP_NMR_comp/ODNP/old/2020",
             "signal",
-            "spincore_Hahn_echoph_v1",
+            "proc_Hahn_echoph",
```

Comment:
why are you pointing to different data?

Reply:
we had moved the data to this folder a while ago and the version is titled wrong as is in the load_data. but we decided to delete this script anyway